naucse/models.py

```diff
@@ -72,7 +73,17 @@ def latex(self):
 
     @reify
     def css(self):
-        return self.info.get('css')
+        css = self.info.get("css")
+
+        if css is None:
+            return None
+
+        parsed = cssutils.parseString(css)
+
+        for rule in parsed.cssRules:
+            rule.selectorText = ".lesson-content " + rule.selectorText
```

Comment:
Can you please add a docstring that explains what happens here?

Reply:
Sure :)